<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-3LAS9CR" failures="2" tests="5" name="failTestScreenShot.ScreenShotTest" time="1.241" errors="3" timestamp="11 Feb 2019 21:33:23 GMT" skipped="0">
  <testcase classname="failTestScreenShot.ScreenShotTest" name="Facebook" time="0.023">
    <error message="Method failTestScreenShot.ScreenShotTest.Facebook() didn&#039;t finish within the time-out 20" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failTestScreenShot.ScreenShotTest.Facebook() didn't finish within the time-out 20
at java.base@10.0.1/java.net.SocketInputStream.socketRead0(Native Method)
at java.base@10.0.1/java.net.SocketInputStream.socketRead(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at app//org.apache.http.impl.conn.LoggingInputStream.read(LoggingInputStream.java:87)
at app//org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:139)
at app//org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:155)
at app//org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:284)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
at app//org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:261)
at app//org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:165)
at app//org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:167)
at app//org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:272)
at app//org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:124)
at app//org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:271)
at app//org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
at app//org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
at app//org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at app//org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
at app//org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
at app//org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
at app//org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
at app//org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:316)
at app//failTestScreenShot.ScreenShotTest.Facebook(ScreenShotTest.java:21)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/java.lang.reflect.Method.invoke(Unknown Source)
at app//org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.base@10.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base@10.0.1/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base@10.0.1/java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- Facebook -->
  <testcase classname="failTestScreenShot.ScreenShotTest" name="Facebook" time="0.026">
    <error message="Method failTestScreenShot.ScreenShotTest.Facebook() didn&#039;t finish within the time-out 20" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failTestScreenShot.ScreenShotTest.Facebook() didn't finish within the time-out 20
at java.base@10.0.1/java.net.SocketInputStream.socketRead0(Native Method)
at java.base@10.0.1/java.net.SocketInputStream.socketRead(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at app//org.apache.http.impl.conn.LoggingInputStream.read(LoggingInputStream.java:87)
at app//org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:139)
at app//org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:155)
at app//org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:284)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
at app//org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:261)
at app//org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:165)
at app//org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:167)
at app//org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:272)
at app//org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:124)
at app//org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:271)
at app//org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
at app//org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
at app//org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at app//org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
at app//org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
at app//org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
at app//org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
at app//org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:316)
at app//failTestScreenShot.ScreenShotTest.Facebook(ScreenShotTest.java:21)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/java.lang.reflect.Method.invoke(Unknown Source)
at app//org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.base@10.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base@10.0.1/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base@10.0.1/java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- Facebook -->
  <testcase classname="failTestScreenShot.ScreenShotTest" name="Facebook" time="0.035">
    <error message="Method failTestScreenShot.ScreenShotTest.Facebook() didn&#039;t finish within the time-out 20" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failTestScreenShot.ScreenShotTest.Facebook() didn't finish within the time-out 20
at java.base@10.0.1/java.net.SocketInputStream.socketRead0(Native Method)
at java.base@10.0.1/java.net.SocketInputStream.socketRead(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at java.base@10.0.1/java.net.SocketInputStream.read(Unknown Source)
at app//org.apache.http.impl.conn.LoggingInputStream.read(LoggingInputStream.java:87)
at app//org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:139)
at app//org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:155)
at app//org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:284)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
at app//org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
at app//org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:261)
at app//org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:165)
at app//org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:167)
at app//org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:272)
at app//org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:124)
at app//org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:271)
at app//org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
at app//org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
at app//org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at app//org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
at app//org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
at app//org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
at app//org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
at app//org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
at app//org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
at app//org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:316)
at app//failTestScreenShot.ScreenShotTest.Facebook(ScreenShotTest.java:21)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@10.0.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base@10.0.1/java.lang.reflect.Method.invoke(Unknown Source)
at app//org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.base@10.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base@10.0.1/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base@10.0.1/java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- Facebook -->
  <testcase classname="failTestScreenShot.ScreenShotTest" name="Google" time="0.863">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at failTestScreenShot.ScreenShotTest.Google(ScreenShotTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Google -->
  <testcase classname="failTestScreenShot.ScreenShotTest" name="SuperTech" time="0.294">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at failTestScreenShot.ScreenShotTest.SuperTech(ScreenShotTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SuperTech -->
</testsuite> <!-- failTestScreenShot.ScreenShotTest -->
